addons/account_peppol/views/res_config_settings_views.xml

```diff
@@ -5,200 +5,198 @@
-                <div attrs="{'invisible': [('is_account_peppol_eligible', '=', False)]}">
-                    <h2>PEPPOL Electronic Document Invoicing</h2>
-                    <div class="row mt16 o_settings_container" id='account_peppol'>
-                        <div class="col-12 col-lg-12 o_setting_box">
-                            <field name="account_peppol_proxy_state" invisible="1"/>
-                            <div class="o_setting_left_pane">
-                                <field name="is_account_peppol_participant"
-                                       attrs="{'invisible': [('account_peppol_proxy_state', '!=', 'not_registered')]}"/>
-                            </div>
-                            <div class="o_setting_right_pane">
-                                <div attrs="{'invisible': [('account_peppol_proxy_state', '!=', 'not_registered')]}">
-                                    <label for="is_account_peppol_participant" string='Use PEPPOL Invoicing'/>
-                                    <span class="fa fa-lg fa-building-o" title="Values set here are company-specific."/>
-                                    <div class="text-muted">
-                                        Our staff members will first review your request within two working days.
-                                    </div>
-                                    <div class="text-muted">
-                                        Start sending and receiving documents via Peppol as soon as you receive our email notification.
+            <block id="analytic" position="after">
+                <block title="PEPPOL Electronic Document Invoicing">
+                    <field name="is_account_peppol_eligible" invisible="1"/>
+                    <div attrs="{'invisible': [('is_account_peppol_eligible', '=', False)]}">
+                        <div id='account_peppol'>
+                            <div class="col-12 col-lg-12 o_setting_box">
+                                <field name="account_peppol_proxy_state" invisible="1"/>
+                                <div class="o_setting_left_pane">
+                                    <field name="is_account_peppol_participant"
+                                        attrs="{'invisible': [('account_peppol_proxy_state', '!=', 'not_registered')]}"/>
+                                </div>
+                                <div class="o_setting_right_pane">
+                                    <div attrs="{'invisible': [('account_peppol_proxy_state', '!=', 'not_registered')]}">
+                                        <label for="is_account_peppol_participant" string='Use PEPPOL Invoicing'/>
+                                        <span class="fa fa-lg fa-building-o" title="Values set here are company-specific."/>
+                                        <div class="text-muted">
+                                            Start sending and receiving documents via Peppol as soon as your registration is complete.
```

Comment:
Since the wording has changed in stable version, you need to update the .pot file accordingly